envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -24,6 +24,12 @@ class SnapshotProperties {
     var enabledCommunicationModes = EnabledCommunicationModes()
     var shouldSendMissingEndpoints = false
     var metrics: MetricsProperties = MetricsProperties()
+    var accessLogFilterProperties: AccessLogFilterProperties = AccessLogFilterProperties()
+}
+
+class AccessLogFilterProperties {
```

Comment:
Why do we need these indexes? Can filter regex be modified somehow?

Reply:
right, we do not need them